main.go

```diff
@@ -1,10 +1,10 @@
 package main
 
 import (
-	"os"
-
 	"github.com/pee2pee/lse/ls"
 	"github.com/spf13/cobra"
+	"os"
+	_ "path/filepath"
```

Comment:
remove unused import `path/filepath`

Reply:
Actually used the filepath's walk function to recursively print the subdirectories

Reply:
The `path/filepath` is used in `ls/ls.go`, not `main.go`, so no need to add it to main.go.

Reply:
Sure. Fixed it:sunglasses: